frontend/src/components/create-podcast/create-podcast.tsx

```diff
@@ -0,0 +1,80 @@
+import { PodcastCreatePayload } from 'common/types/types';
+import { DataStatus } from 'common/enums/enums';
+import { podcast as podcastActions } from 'store/actions';
+import { useAppSelector, useDispatch, useState } from 'hooks/hooks';
+import { Input, Button } from 'components/common/common';
+import styles from './styles.module.scss';
+
+const CreatePodcast: React.FC = () => {
+  const [podcastCreatePayload, setPodcastCreatePayload] =
+    useState<PodcastCreatePayload>({
```

Comment:
```suggestion
    useState<PodcastCreatePayload>({
```
use useForm hook for all forms